modules/ROOT/pages/_partials/sync/server-data-access.adoc

```diff
@@ -1,3 +1,125 @@
 = Data Access
 
-TODO
\ No newline at end of file
+It is very likely a {sync-service} application will need to access data from a database for it to work. The GraphQL spec or Voyager libraries does not require an application to use a particular type of database, and the developer of the application can choose a database that is most suitable for the application. 
```

Comment:
Is the point here that:

Typically when working with GraphQL, you use a database  to store data, for example, MongoDB or PostgreSQL.
To summarize the tasks associated with using a database from {sync-service}:

Reply:
kind of. My main point here is that the develop can choose whatever database that's suits their app.

Reply:
does mentioning Voyager in this context introduce the possibility that it doesn't work with sync service, ie will users care that it works with Voyager?

Reply:
I don't think so. I think users do care, because some sync libraries work with only some type of database. For example, the old sync only works with mongodb. CouchDB only works with CouchDB etc. But with Voyager, you can use any.